includes/class-wc-stripe-checkout-sessions-controller.php

```
@@ -68,12 +70,37 @@ public function create_checkout_session(): void {
 				];
 			}
 
+			$first_name = get_user_meta( $user_id, 'first_name', true );
```

PHPMD (2.15.0)

[error] The variable $first_name is not named in camelCase. (undefined)

(CamelCaseVariableName)